hedera-mirror-rest/api/v1/openapi.yml

```
@@ -1929,6 +1998,13 @@ components:
         oneOf:
           - $ref: '#/components/schemas/EntityId'
           - $ref: '#/components/schemas/SolidityAddress'
+    indexQueryParam:
```

nit: just to be clear
```suggestion
    logIndexQueryParam:
```

---

Fixed